### DISCUSSION TITLE
The run_files directory is not detected.  Please run the preparation on the ISCE stack....

### DISCUSSION THREAD

**Discussion by AllendePrieto** [alexisInSAR/EZ-InSAR#31] [Help]
Hi,
I want to congratulate you on the development of this tool. I have managed to follow the installation and execution steps, but now I can't continue.
In the "Pre-run of ISCE processing" step, I entered the required parameters, but the "configs" and "run_files" directories were not generated.
![image](https://user-images.githubusercontent.com/131383377/233412960-ef0b1be6-6727-486c-9eae-463d0e95cc8f.png)
![image](https://user-images.githubusercontent.com/131383377/233414515-368f4b7d-b4fd-4f98-addb-18fb176150e6.png)
Can you help me please?
Thanks

**Comment by alexisInSAR** [comment]
Hi, 

Thank you! 

You need to install the "cv2" package. Please, in your conda environment (after the command _load_insar_), please run this command: 
````
conda install opencv-python
````  

It should be fine. 

Alexis

**Reply by AllendePrieto** [reply]
Perfect!!  Thanks!!

**Comment by junaidlonez11** [comment]
Hi @alexisInSAR 

I also face the same issue, "The run_files directory is not detected. Please run the preparation of the ISCE stack."

 I have openCv 4.7.0 installed 
(InSARenv) jlone@IF-PCC-FHLCZC3:~$ python -c "import cv2; print(cv2.__version__)"
4.7.0


![image](https://github.com/alexisInSAR/EZ-InSAR/assets/57167285/720e17c5-096f-4bff-896d-b75da0e15841)


Any suggestion is much appreciated.
Thank you!

**Reply by Rick0219** [reply]
I've looked at your error report, it should be an error when downloading the orbites file, I suggest you download the precision orbits file and put them in the orbit folder
